interview-questions.md

```diff
@@ -7,50 +7,50 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 INSTRUCTOR EXAMPLE: What is a conditional statement?
 
-  Your answer:
+  Your answer: A conditional statement is a function that performs an action based on the results of a Boolean statement. You create a decision tree that checks if a condition is true or false. If true, the statement will perform a programmed action. If it is not true, then it will perform a different programmed action.
 
-  Researched answer:
+  Researched answer: My research shows that my answer was correct.
 
 
 
 1. What is git? What is the difference between git and Github?
 
-  Your answer:
+  Your answer: Git is the software used by developers to collaborate and share code. GitHub is the website that allows a developer to interact with the remote files in a GUI. For my coding bootcamp, I would regularly pull repositories from GitHub and push my local changes back up.
 
-  Researched answer:
+  Researched answer: Git is software used by developers on their local machine to manage multiple versions of files. GitHub is a website that serves as a cloud repository for git. GitHub users can manage remote files using a GUI.
 
 
 
 2. Which JavaScript operators will return a Boolean value?
 
-  Your answer:
+  Your answer: The operators that will return a Boolean value are those that will return a true/false statement. That would be relational operators (>, <, >=, <=), logical operators (&&, ||, !), and equivalent operators (==, ===).
 
-  Researched answer:
+  Researched answer: I forgot modulo (%). I've commonly used modulo to determine if a value is even or not.
+  Operators that return a Boolean value are often used in conditional statements.
 
 
 
 3. What is an index? What is the difference between index and value?
 
-  Your answer:
+  Your answer: Index is the location of a character in a string or an element in an array. The measure of an index begins at the 0 position with the first character or element. A value is a piece of data that is assigned meaning.
 
-  Researched answer:
+  Researched answer: My research shows my answer is correct.
```

Comment:
Okay something to know: I will not allow this answer moving forward. Correctness is not the skill we are here assessing. Preparedness for interviews and the ability to grow your knowledge when prompted is what is being assessed for. The question is not "Were you close?" the question is "What else can you find? How good are you at research?"